src/k8s-engine/wsl.ts

```diff
@@ -1207,8 +1230,25 @@ export default class WSLBackend extends events.EventEmitter implements K8s.Kuber
             const rotateConf = LOGROTATE_K3S_SCRIPT.replace(/\r/g, '')
               .replace('/var/log', logPath);
 
+            if (this.cfg?.experimentalHostResolver) {
+              console.log(`launching experimental DNS host-resolver`);
```

Comment:
```suggestion
              console.debug(`Launching experimental DNS host-resolver`);
```
We probably don't need this logged unless the user ticked the debug mode checkbox in Troubleshooting.